quinn-h3/src/client.rs

```diff
@@ -306,11 +306,7 @@ impl RecvResponse {
     }
 
     pub fn body(self) -> RecvBody {
-        RecvBody::new(
-            self.recv,
-            self.conn.clone(),
-            self.stream_id,
-        )
+        RecvBody::new(self.recv, self.conn.clone(), self.stream_id)
```

Comment:
Some of the changes in this commit seem unrelated and due to reformatting, could you please isolate them in a separate commit?

Reply:
Done in the previous commit.